aws/resource_aws_vpc_dhcp_options_test.go

```diff
@@ -11,6 +12,82 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
+func init() {
+	resource.AddTestSweepers("aws_vpc_dhcp_options", &resource.Sweeper{
+		Name: "aws_vpc_dhcp_options",
+		F:    testSweepVpcDhcpOptions,
+	})
+}
+
+func testSweepVpcDhcpOptions(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %s", err)
+	}
+	conn := client.(*AWSClient).ec2conn
+
+	input := &ec2.DescribeDhcpOptionsInput{}
+
+	err = conn.DescribeDhcpOptionsPages(input, func(page *ec2.DescribeDhcpOptionsOutput, lastPage bool) bool {
+		for _, dhcpOption := range page.DhcpOptions {
+			var domainName string
+			var defaultDomainNameFound, defaultDomainNameServersFound bool
+
+			if region == "us-east-1" {
+				domainName = "ec2.internal"
+			} else {
+				domainName = region + ".compute.internal"
+			}
+
+			for _, dhcpConfiguration := range dhcpOption.DhcpConfigurations {
+				if aws.StringValue(dhcpConfiguration.Key) == "domain-name" {
+					if len(dhcpConfiguration.Values) == 0 || len(dhcpConfiguration.Values) > 1 {
+						continue
+					}
+
+					if dhcpConfiguration.Values[0] != nil && aws.StringValue(dhcpConfiguration.Values[0].Value) == domainName {
```

Comment:
After applying the above you can change this to the following
```suggestion
					if aws.StringValue(dhcpConfiguration.Values[0].Value) == domainName {
```